recipes/propelauth_py/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "propelauth_py" %}
+{% set version = "4.2.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/propelauth-py-{{ version }}.tar.gz
+  sha256: 746d1c3cc0117d2d57dd03da3513f6e0d3607cd80580b537549b9058755fe244
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+  run:
+    - python >={{ python_min }}
+    - pyjwt >=2,<3
```

Comment:
Note that upstream specifies `pyjwt[crytpro]`, which adds an additional requirement:
https://github.com/jpadilla/pyjwt/blob/master/pyproject.toml#L45-L47

Reply:
cryptography added - Thank you @synapticarbors